modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -372,7 +372,7 @@ public String extractIndexableAttributes(
 
 				Serializable value = field.getValue(locale);
 
-				if (Validator.isNull(value)) {
+				if (Validator.isBlank(String.valueOf(value))) {
```

Comment:
@noornajj I think you also need to check if `value == null` here.. There are tests failing that seem to be related to this.

```
if ((value == null) || Validator.isBlank(String.valueOf(value))) {
```

Reply:
@carolmariaabb Woulnd'y it make more sense to go back to the original condition?
`if ((value == null) || Objects.equals(value, StringPool.BLANK)) {`

Reply:
@noornajj `Validator.isBlank(String.valueOf(value)) == Objects.equals(value, StringPool.BLANK)` but is different from  `(value == null)`

Reply:
Yes, I know `(value == null)` is different. I was just wondering if there's a preference between `Validator.isBlank(String.valueOf(value))` and `Objects.equals(value, StringPool.BLANK)`

Reply:
@noornajj The preference is because you are already using `Validation.isBlank` elsewhere... so it would follow the same pattern.